plugins/ssh-keys/functions

```diff
@@ -12,10 +12,12 @@ list_ssh_keys() {
 verify_ssh_key_file() {
   declare desc="Test that public key is valid"
   [[ -s ${DOKKU_ROOT}/.ssh/authorized_keys ]] || dokku_log_fail "No public keys found."
-  local key line=0
+  local key line=0 tmp_key_file="$(mktemp)"
+  trap "rm \"$tmp_key_file\"" EXIT
```

Comment:
I *think* you can avoid the extra quotes and it'll work as desired, right @michaelshobbs ?